### ISSUE TITLE
Demo is not working

### ISSUE THREAD

**Issue by jasondavis** [podio/jquery-mentions-input#147]
For the past month or more I have noticed the demo

http://podio.github.io/jquery-mentions-input/

Does not work.  It seems one of the issues...

    Failed to load resource: the server responded with a status of 404 (Not Found)

**Comment by leizard** [comment]
yeah. but does the package work for you ? I give it a try and it doesn't work as well ( like the demo )

**Comment by leizard** [comment]
I figure it out. that because the script in http://documentcloud.github.io/underscore/underscore-min.js not available anymore

**Comment by ericyau82** [comment]
I face the same issue so I download the missing js file* from:
http://underscorejs.org/
*file:
http://underscorejs.org/underscore-min.js
and test the demo on local

**Comment by Gaffney** [comment]
I noticed 27 days ago a fix was put up for this (although this was not closed).  The demo is still broken.

```javascript
Uncaught ReferenceError: _ is not defined
examples.js:3 Uncaught TypeError: $(...).mentionsInput is not a function```
```

**Comment by rept** [comment]
Indeed still broken.
jquery.mentionsInput.js:387 Uncaught ReferenceError: _ is not defined

**Comment by timdiggins** [comment]
Seems sad to have a demo and it be broken. Much worse than not having a demo. Would you welcome a PR to remove (even temporarily) the link to the demo?

Also, (OT) think that it might be worth just closing the 3 antique open PRs

**Comment by praveenscience** [comment]
This has been fixed and can be closed. :+1:

**Comment by sajjadafridi** [comment]
Replace the js version to 2.2.4
